### PR TITLE
Media files

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: coss/settings.py [hunk 1]
```diff
@@ -45,7 +45,14 @@
 
 STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
 
-MEDIA_URL = config('MEDIA_URL', '/media/')
+# S3 storages
+AWS_STORAGE_BUCKET_NAME = config('AWS_STORAGE_BUCKET_NAME', default='')
+AWS_LOCATION = config('AWS_LOCATION', default='cms')
```
Comment (akatsoulas) [inline coss/settings.py:50]: Why do you need this? Or better asked, how would you like to use this?

File: coss/settings.py [hunk 2]
```diff
@@ -45,7 +45,14 @@
...
+AWS_S3_CUSTOM_DOMAIN = config('AWS_S3_CUSTOM_DOMAIN',
+                              default='{0}.s3.amazonaws.com'.format(AWS_STORAGE_BUCKET_NAME))
+
+MEDIA_URL = config('MEDIA_URL', default='https://{0}/'.format(AWS_S3_CUSTOM_DOMAIN))
+DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
```
Comment (akatsoulas) [inline coss/settings.py:55]: What about the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY envs?

**Review by akatsoulas — APPROVED** [review]
r+wc

**Pull request merged**